### PR TITLE
Shroud and Charge BKB fix

### PR DESCRIPTION
**Pull request by imaGecko**
I think this is all?

### CODE REVIEW

File: game/scripts/vscripts/items/charge_bkb.lua [hunk 1]
```diff
@@ -97,3 +97,7 @@ function modifier_charge_bkb_on_created(keys)
 --      return 10.0
 --   end)
 end
+
+item_charge_bkb_2 = item_charge_bkb
```
Comment (chrisinajar) [inline game/scripts/vscripts/items/charge_bkb.lua]: There is no item_charge_bkb.

This item is data driven and i don't really know how it works

File: game/scripts/vscripts/items/shroud.lua [hunk 2]
```diff
@@ -25,6 +25,14 @@ item_shroud_2 = item_shroud --luacheck: ignore item_shroud_2
 
 --------------------------------------------------------------------------------
 
+item_shroud_3 = item_shroud --luacheck: ignore item_shroud_3
```
Comment (chrisinajar) [inline game/scripts/vscripts/items/shroud.lua]: i don't think you need the luacheck ignore lines

File: game/scripts/vscripts/items/charge_bkb.lua (hunk 1 above)
Comment (Minnakht) [inline game/scripts/vscripts/items/charge_bkb.lua]: at this point I have my doubts if anyone does, then.

...is a lua rewrite in order?

Comment (imaGecko) [inline game/scripts/vscripts/items/charge_bkb.lua]: and I thought I was being smart for once

**Pull request merged**